orb/orb.yml

```diff
@@ -98,13 +98,18 @@ jobs:
 
 commands:
   use-secrets:
-    description: Will add the secrets to BASH_ENV
+    description: Will add the secrets to BASH_ENV, defaults to /tmp/secrets
+    parameters:
+      file:
+        type: string
+        default: "/tmp/secrets"
+        description: filepath to source (defaults to /tmp/secrets)
```

Comment:
Nitpick: Capital F in Filepath to be consistent with the other description attributes.

Reply:
Fixed